examples/slack_alert/config.yaml

```diff
@@ -0,0 +1,22 @@
+name: post_pipeline
```

Comment:
for other examples you mentioned in the README the option to run with `-c config`, why not for Slack?